requirements-testing.txt

```diff
@@ -1,2 +1,5 @@
 numpy<1.19.4
+https://extras.wxpython.org/wxPython4/extras/linux/gtk3/ubuntu-16.04/wxPython-4.1.0-cp36-cp36m-linux_x86_64.whl
```

Comment:
This bends thie particular requirements file into a what-Travis-needs spec, which I think is fine. In the past, we have used it to set up local environments, so it was a bit more platform-independent. We recently did some cleanup on setup.py to add the `dev` and `memprof` extras, which made the requirements files pretty much vestigial, so I don't think anyone uses it to set up their own environment anymore. Might be nice to just add a comment to the top that says something like "this requirements file is mostly here to make Travis CI go"

Reply:
Ok, I'll add a comment to the top of this file.